src/loaders/api/api.js

```diff
@@ -61,13 +61,23 @@ export function setAPI (agentIdentifier, nr, forceDrain) {
     if (typeof name !== 'string') return
     if (name.charAt(0) !== '/') name = '/' + name
     getRuntime(agentIdentifier).customTransaction = (host || 'http://custom.transaction') + name
-    return apiCall(prefix, 'setPageViewName', true, 'api')()
+    return apiCall(prefix, 'setPageViewName', true)()
   }
 
-  nr.setCustomAttribute = function (name, value) {
+  function appendJsAttribute (key, value, apiName = 'setCustomAttribute') {
     const currentInfo = getInfo(agentIdentifier)
-    setInfo(agentIdentifier, { ...currentInfo, jsAttributes: { ...currentInfo.jsAttributes, [name]: value } })
-    return apiCall(prefix, 'setCustomAttribute', true, 'api')()
+    setInfo(agentIdentifier, { ...currentInfo, jsAttributes: { ...currentInfo.jsAttributes, [key]: value } })
+    return apiCall(prefix, apiName, true)()
```

Comment:
just to confirm, was `api` buffer group removed because there is no async consumer of the ee buffer?

Reply:
Righto